config/global.json

```diff
@@ -68,5 +62,71 @@
     "Permit2Proxy",
     "TokenWrapper"
   ],
-  "blacklistedFunctionSelectors": ["0x23b872dd"]
+  "blacklistedFunctionSelectors": ["0x23b872dd"],
+  "whitelistPeripherySelectors": {
```

Comment:
this map also include signatures so perhaps naming it `whitelistPeripheryFunctions` would be better

Reply:
thanks, updated